### ISSUE TITLE
java.lang.Throwable: Invalid root file

### ISSUE THREAD

**Issue by peterlin741** [GoogleCloudPlatform/cloud-code-intellij#2817]
```
java.lang.Throwable: Invalid root file:///Users/[user_file_path_redacted] in Module: 'prebuilt-platform.domain'
	at com.intellij.openapi.diagnostic.Logger.error(Logger.java:143)
	at com.intellij.openapi.roots.impl.RootIndex.ensureValid(RootIndex.java:319)
	at com.intellij.openapi.roots.impl.RootIndex.ensureValid(RootIndex.java:306)
	at com.intellij.openapi.roots.impl.RootIndex.buildRootInfo(RootIndex.java:155)
	at com.intellij.openapi.roots.impl.RootIndex.<init>(RootIndex.java:66)
	at com.intellij.openapi.roots.impl.DirectoryIndexImpl.getRootIndex(DirectoryIndexImpl.java:134)
	at com.intellij.openapi.roots.impl.DirectoryIndexImpl.getInfoForFile(DirectoryIndexImpl.java:147)
	at com.intellij.openapi.roots.impl.FileIndexBase.getInfoForFileOrDirectory(FileIndexBase.java:105)
	at com.intellij.openapi.roots.impl.ModuleFileIndexImpl.getInfoForFileOrDirectory(ModuleFileIndexImpl.java:20)
	at com.intellij.openapi.roots.impl.ModuleFileIndexImpl.lambda$getModuleRootsToIterate$0(ModuleFileIndexImpl.java:54)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:865)
	at com.intellij.openapi.application.ReadAction.compute(ReadAction.java:61)
	at com.intellij.openapi.roots.impl.ModuleFileIndexImpl.getModuleRootsToIterate(ModuleFileIndexImpl.java:45)
	at com.intellij.openapi.roots.impl.ModuleFileIndexImpl.iterateContent(ModuleFileIndexImpl.java:33)
	at com.intellij.util.indexing.roots.ModuleIndexableFilesProvider.iterateFiles(ModuleIndexableFilesProvider.kt:32)
	at com.intellij.util.indexing.FileBasedIndexEx.iterateIndexableFiles(FileBasedIndexEx.java:375)
	at com.intellij.util.indexing.FileBasedIndexImpl.projectIndexableFiles(FileBasedIndexImpl.java:794)
	at com.intellij.util.indexing.FileBasedIndexEx.processValuesInScope(FileBasedIndexEx.java:286)
	at com.intellij.util.indexing.FileBasedIndexEx.getContainingFiles(FileBasedIndexEx.java:167)
	at com.intellij.psi.search.FileTypeIndex.getFiles(FileTypeIndex.java:39)
	at com.google.cloud.tools.intellij.kubernetes.skaffold.SkaffoldFileService$findSkaffoldFiles$1.compute(SkaffoldFileService.kt:106)
	at com.google.cloud.tools.intellij.kubernetes.skaffold.SkaffoldFileService$findSkaffoldFiles$1.compute(SkaffoldFileService.kt:39)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:855)
	at com.google.cloud.tools.intellij.kubernetes.skaffold.SkaffoldFileService.findSkaffoldFiles(SkaffoldFileService.kt:105)
	at com.google.cloud.tools.intellij.kubernetes.skaffold.SkaffoldConfigurationDetector.showPromptForSkaffoldConfigurations(SkaffoldConfigurationDetector.kt:126)
	at com.google.cloud.tools.intellij.kubernetes.skaffold.init.SkaffoldInitGradleSyncListener.onSuccess(SkaffoldInitGradleSyncListener.kt:41)
	at com.intellij.openapi.externalSystem.service.remote.ExternalSystemProgressNotificationManagerImpl$onSuccess$1.invoke(ExternalSystemProgressNotificationManagerImpl.kt:59)
	at com.intellij.openapi.externalSystem.service.remote.ExternalSystemProgressNotificationManagerImpl$onSuccess$1.invoke(ExternalSystemProgressNotificationManagerImpl.kt:15)
	at com.intellij.openapi.externalSystem.service.remote.ExternalSystemProgressNotificationManagerImpl$forEachListener$1.invoke(ExternalSystemProgressNotificationManagerImpl.kt:88)
	at com.intellij.openapi.externalSystem.service.remote.ExternalSystemProgressNotificationManagerImpl$forEachListener$1.invoke(ExternalSystemProgressNotificationManagerImpl.kt:15)
	at com.intellij.openapi.externalSystem.service.remote.ExternalSystemProgressNotificationManagerImpl$sam$java_util_function_Consumer$0.accept(ExternalSystemProgressNotificationManagerImpl.kt)
	at com.intellij.openapi.extensions.impl.ExtensionProcessingHelper.forEachExtensionSafe(ExtensionProcessingHelper.java:33)
	at com.intellij.openapi.extensions.ExtensionPointName.forEachExtensionSafe(ExtensionPointName.java:49)
	at com.intellij.openapi.externalSystem.service.remote.ExternalSystemProgressNotificationManagerImpl.forEachListener(ExternalSystemProgressNotificationManagerImpl.kt:88)
	at com.intellij.openapi.externalSystem.service.remote.ExternalSystemProgressNotificationManagerImpl.onSuccess(ExternalSystemProgressNotificationManagerImpl.kt:59)
	at com.intellij.openapi.externalSystem.service.internal.ExternalSystemResolveProjectTask.doExecute(ExternalSystemResolveProjectTask.java:121)
	at com.intellij.openapi.externalSystem.service.internal.AbstractExternalSystemTask.execute(AbstractExternalSystemTask.java:147)
	at com.intellij.openapi.externalSystem.service.internal.AbstractExternalSystemTask.execute(AbstractExternalSystemTask.java:133)
	at com.intellij.openapi.externalSystem.util.ExternalSystemUtil$2.executeImpl(ExternalSystemUtil.java:519)
	at com.intellij.openapi.externalSystem.util.ExternalSystemUtil$2.lambda$execute$1(ExternalSystemUtil.java:372)
	at com.intellij.openapi.project.DumbServiceHeavyActivities.suspendIndexingAndRun(DumbServiceHeavyActivities.java:20)
	at com.intellij.openapi.project.DumbServiceImpl.suspendIndexingAndRun(DumbServiceImpl.java:153)
	at com.intellij.openapi.externalSystem.util.ExternalSystemUtil$2.execute(ExternalSystemUtil.java:372)
	at com.intellij.openapi.externalSystem.util.ExternalSystemUtil$4.run(ExternalSystemUtil.java:627)
	at com.intellij.openapi.progress.impl.CoreProgressManager$TaskRunnable.run(CoreProgressManager.java:935)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcessWithProgressAsync$5(CoreProgressManager.java:442)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$3(ProgressRunner.java:235)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:170)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:629)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:581)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:60)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:157)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$4(ProgressRunner.java:235)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1700)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:668)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:665)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:665)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

### Additional Information

- IDE type (e.g. IntelliJ, Pycharm): IntelliJ
- IDE version: 2020.2
- Cloud Code version (Settings > Plugins > Cloud Code):  20.6.1-202
- Operating System: Mac OS X

**Comment by jmah-cc** [comment]
This has been fixed, will be available in the next release 20.12.2

**Comment by ivanporty** [comment]
https://listnrcrash.corp.google.com/product/717949/report/1000001108940112166
Fresh report from PyCharm, latest version
